### ISSUE TITLE
integrate BSI ot-parsers

### ISSUE THREAD

**Issue by mmguero** [cisagov/Malcolm#456]
**[<img src="https://avatars.githubusercontent.com/u/13872653?s=17&v=4" width="17" height="17"> @mmguero](https://github.com/mmguero)** cloned issue [idaholab/Malcolm#327](https://github.com/idaholab/Malcolm/issues/327) on 2024-01-09: 

> [DINA-community/ot-parsers](https://github.com/DINA-community/ot-parsers)
> 
> 
> 
> Examine these parsers and pull some of them into Malcolm.

**Comment by Crubumble** [comment]
@mmguero Should be add here, every repository that is ready for integration or should we open an issue for every zeek-plugin?

**Comment by mmguero** [comment]
IMO you could just link the repositories here and we could review them, @piercema what do you think?

**Comment by piercema** [comment]
I agree with that. Link the repositories that are ready here and we'll work on reviewing them

**Comment by Crubumble** [comment]
I would start with a rare/special one [IEC61850 Sampled Value](https://github.com/DINA-community/icsnpp-iec61850-sv)

It would help us a lot if it would be part of Malcolm. That makes it for us easier to convince operator to use and improve it. 

last building test was with

Found Zeek:            yes
Version:               7.2.2 (70202)
Build type:            release

Found Spicy:           yes
Version:               1.13.2 (11302)
Build type:            release
Have toolchain:        yes

**Comment by Crubumble** [comment]
The parser is now running on Zeek 8. However, the number of log entries generated by the protocol is not realistic to process. Therefore, further adjustments must be made (by our side).
